CHANGELOG.md

```diff
@@ -6,78 +6,89 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+- Offers - make, take, and share your offers.
+- Integrated light wallet sync - to get you synced up faster while your full node syncs.
+- Wallet mode - Access just the wallet features to make and receive transactions.
+- Farmer mode - All your farming tools, and full node, while getting all the benefits of the upgraded wallet features.
+- New v2 DB - improved compression for smaller footprint (the v2 DB is created alongside the v1 DB. Please be sure to have enough disk space before executing the DB upgrade command).
+- Key derivation tool via CLI - lets you derive wallet addresses, child keys, and also search your keys for arbitrary wallet addresses/keys.
+- Light wallet data migration - CAT wallets you set up and your offer history will be carried over.
+- The farmer will report version info in User-Agent field for pool protocol (Thanks @FazendaPool).
+- Added new RPC, get_version, to the daemon to return the version of Chia (Thanks @dkackman).
 - Added new config.yaml setting, reserved_cores, to specify how many cores Chia will not use when launching process pools. Using 0 will allow Chia to use all cores for process pools. Set the default to 0 to allow Chia to use all cores. This can result in faster syncing and better performance overall especially on lower-end CPUs like the Raspberry Pi4.
 - Added new RPC, get_logged_in_fingerprint, to the wallet to return the currently logged in fingerprint.
 - Added new CLI option, chia keys derive, to allow deriving any number of keys in various ways. This is particularly useful to do an exhaustive search for a given address using chia keys derive search.
-- Div soft fork block height set to 2,300,000
+- Div soft fork block height set to 2,300,000.
+- Added the ability to add an optional fee for creating and changing plot NFTs.
+- Added *multiprocessing_start_method:* entry in config.yaml that allows setting the python *start method* for multiprocessing (default is *spawn* on Windows & MacOS, *fork* on Unix).
+- Added option to "Cancel transaction" accepted offers that are stuck in "pending".
 
 ### Changed:
 
-- Light wallet client sync updated to only require 3 peers instead of 5
-- Only CATs from the default CAT list will be automatically added, all other unknown CATs will need to be manually added
-- New sorting pattern for offer history - Open/pending offers sorted on top ordered by creation date > confirmation block height > trade id, and then Confirmed and Cancelled offers sorted by the same order
-- When plotting multiple plots with the GUI, new items are taken from the top of the list instead of the bottom
-- CA certificate store update
-- VDF, chiapos, and blspy workflows updated to support python 3.10 wheels
+- Light wallet client sync updated to only require 3 peers instead of 5.
+- Only CATs from the default CAT list will be automatically added, all other unknown CATs will need to be manually added (this behavior can be toggled in config.yaml).
```

Comment:
Might be worth crediting ojura for part of this (specifically the config.yaml toggle)
https://github.com/Chia-Network/chia-blockchain/commit/05f96670181654415b1ad219d38463228f30b0ce

Reply:
Added